interchain-security/dungeon-1/README.md

```diff
@@ -0,0 +1,78 @@
+
+# `dungeon` Chain Details
+
+> Status: **PENDING**
+
+The `dungeon` chain is launched to test the dungeon chain.
+
+---
+
+<!--
+  *Compress the dungeond file (due to size)*
+  cp $(which dungeond) ./dungeond-linux-amd64
+  shasum -a 256 dungeond-linux-amd64
+  tar -czvf dungeond-linux-amd64.tar.gz dungeond-linux-amd64
+  rm ./dungeond-linux-amd64
+
+  *Get CCV Only from genesis*
+  cat genesis.json | jq .app_state.ccvconsumer > dungeon-ccv.json
+
+  *Genesis Without CCV Consumer*
+  cp genesis.json tmp-without-ccv.json
+  jq 'del(.app_state.ccvconsumer)' tmp-without-ccv.json > dungeon-genesis-without-ccv.json
+  rm tmp-without-ccv.json genesis.json
+
+  *Compress Dungeon Gneisis Without CCV*
+  tar -czvf dungeon-genesis-without-ccv.json dungeon-genesis-without-ccv.json
+-->
+
+
+* **Chain-ID**: `dungeon-1`
+* **denom**: `udgn`
+* **Spawn time**: `2024-08-28T13:30:00.000000Z`
+* **Stop time**: `-`
+* **GitHub repo**: [Crypto-Dungeon/dungeonchain](https://github.com/Crypto-Dungeon/dungeonchain)
+* **Commit**: [`f2325ed0e0d341f4280992ac3cbc61b9a967d578`](https://github.com/Crypto-Dungeon/dungeonchain/commit/f2325ed0e0d341f4280992ac3cbc61b9a967d578)
+* **Genesis file with CCV state:** [genesis.json.tar.gz](genesis.json.tar.gz)
+  * `tar -xvzf genesis.json.tar.gz "./network/dungeon-1/genesis.json" --one-top-level=genesis.json --strip-components 4`
+
+* **Reference binary**: `dungeond-linux-amd64`
+  * ` tar -xvzf dungeond-linux-amd64.tar.gz`
+* **Binary sha256sum**: `e44429f3f67c2c57ced7be208092264aed712e6d7b4140e03878552d50dbb2e8`
+* **Genesis file _without CCV state_:** `dungeon-genesis-without-ccv.json`
```

Comment:
The airdrop of accounts is done at genesis is 120mb, which is above Githubs file size limit of 100mb. We have to compress since 250k+ accounts got the airdrop

Reply:
I see. Is this to match the accounts that will be listed in the mainnet genesis file?

Reply:
yes, its an exact 1-1 of mainnet state. Xan promised people they could see their balances in keplr before mainnet.